receiver/bigipreceiver/testdata/config.yaml

```diff
@@ -0,0 +1,19 @@
+receivers:
+  bigip:
+    endpoint: http://localhost:443
+    username: otelu
+    password: $RABBITMQ_PASSWORD
+    collection_interval: 10s
+
+processors:
+  nop:
+
+exporters:
+  nop:
+
+service:
+  pipelines:
+    metrics:
+      receivers: [rabbitmq]
```

Comment:
```suggestion
      receivers: [bigip]
```

Reply:
@BinaryFissionGames Deleted this file as it was not needed